com.elsevier.MendeleyDesktop.appdata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Better description and screenshots would be useful.

Reply:
Done

Reply:
Since the description is taken from their website: Does Mendeley release these snippets under CC0? Going by Elsevier's reputation, probably not...